nautobot/extras/templates/extras/staticgroup_retrieve.html

```diff
@@ -50,6 +50,12 @@
                 <td>Tenant</td>
                 <td>{{ object.tenant|hyperlinked_object }}</td>
             </tr>
+            {% if object.hidden %}
+                <tr>
+                    <td>Hidden</td>
+                    <td>{{ object.hidden|render_boolean }}</td>
+                </tr>
+            {% endif %}
```

Comment:
Why do we only display it when `object.hidden` is true?

Reply:
My thinking was that most/all of the groups that the user actually cares about are non-hidden ones, and you have to go out of your way to even see hidden ones, so the presence of this field in the detail view in the default case wouldn't really add any meaning/value to the average user.